components/console_cmd_mqtt/examples/ssl_mutual_auth/main/ssl_mutual_auth.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Unlicense OR CC0-1.0
+ */
+#include <stdio.h>
+#include "nvs_flash.h"
+#include "esp_netif.h"
+#include "esp_event.h"
+#include <netdb.h>
+#include "console_mqtt.h"
+#include "protocol_examples_common.h"
+
+// Certs for mqtts://test.mosquitto.org:8884
+extern const uint8_t client_cert_pem_start[] asm("_binary_client_crt_start");
```

Comment:
```suggestion
extern const uint8_t g_client_cert_pem_start[] asm("_binary_client_crt_start");
```
some people might be confused, as another static `client_cert_pem_start` appears in the component...